.github/workflows/documentation.yml

```diff
@@ -0,0 +1,27 @@
+name: Generate terraform docs
+on: [push, pull_request]
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.ref }}
+
+    - name: Render terraform docs inside the README.md and push changes back to PR branch
+      uses: terraform-docs/gh-actions@main
+      with:
+        working-dir: .
+        template: |-
```

Comment:
Is this needed, since the `output-file` is specified. See [here](https://github.com/terraform-docs/gh-actions#usage). I think as long as a section like `<!-- BEGIN_TF_DOCS --> and <!-- END_TF_DOCS -->` exists in the `README.md` terraform-docs will automatically update the `README.md` inside of that section.

Reply:
Not really. Thanks :). Just trying to make it fancy. I put the change already in the readme markdown file.